projects/baby_driver/scripts/can_util.py

```diff
@@ -183,3 +183,35 @@ def next_message(
         ))
 
     return msg
+
+def can_pack(data_list):
+    """
+    Converts list of tuples in form ((int) val, (int) len_in_bytes)
+            and combines them into a bytearray rendition
+    Returns:
+        A bytearray object representing message components passed in
+    Raises:
+        ValueError: if tuple has more than 2 items
+        ValueError: if tuple value exceeds allotted length in bytes
+    """
+    to_bytes = ""
+    for item in data_list:
+        if len(item) > 2:
+            raise ValueError("Tuple length {}: expected 2".format(len(item)))
+        #store value and number of bytes allotted
+        val = item[0]
+        len_in_bytes = item[1]
```

Comment:
Python tip: you can just do `for val, len_in_bytes in data_list:`, it'll unpack the tuples automatically.